scss/techs.scss

```diff
@@ -12,6 +12,12 @@ $card-width: 210px;
 .main-tech-container {
   position: relative;
   margin-right: 30px;
+  &:hover {
+    .background-tech-container {
+      left: 10px;
+      top: 10px;
+    }
+  }
 }
 
 .background-tech-container {
```

Comment:
I forgot to mention you have to sort class properties alphabetically. You can install a plugin that highlights those properties that are not sorted. I don't pretend you modify all code, but keep it in mind :smile: